### ISSUE TITLE
Switch to Vello and Glazier

### ISSUE THREAD

**Issue by PoignardAzur** [PoignardAzur/masonry-rs#24]
Vello and Glazier are two successor projects to dependencies Masonry uses, respectively Piet and druid-shell. Both of these dependencies are unlikely to to be maintained in the near future (especially druid-shell), while Vello and Glazier are actively worked on by the Druid community.

As both these projects have APIs very similar to the ones they're succeeding, the switch should be a relatively straightforward affair (famous last words). It should not impact the semantics of Masonry.

Note: If I understood correctly, https://github.com/linebender/xilem/commit/2193b87212605be53c10d3fe3650be9c0e8f0457 is where this was implemented in Xilem.

**Comment by PoignardAzur** [comment]
(Also, once this is done, we should try to remove any remaining references to Druid from the code.)

**Comment by PoignardAzur** [comment]
(Also, remove all references to Piet)

**Comment by RagibHasin** [comment]
Is there any ongoing work on this front? If not, I would be happy to take on this, as a learning opportunity of course.

**Comment by PoignardAzur** [comment]
Nobody is working on this, and I'd definitely appreciate help.

Tell me if you want any help/advice/mentoring.

**Comment by mauvray** [comment]
Maybe I am a bit confused, but in the latest news of xilem, it was said to move to winit instead of glazier. In the end, should the target be glazier or winit or both (with a feature flag ?)

**Comment by PoignardAzur** [comment]
I'm actively working on this issue, though I'm posting more updates on Zulip than here. Tl;dr is yes, we've switched tracks and I'm now porting to Winit. I'll update the issue description.

**Comment by PoignardAzur** [comment]
Actually, I think I'll just close this issue. I may or may not create a new issue with the details of the Winit port.

**Comment by PoignardAzur** [comment]
See https://github.com/PoignardAzur/masonry-rs/pull/56/ for progress.
